GitHub/workflows

187jjay187 opened the pull request:
In This Pull Request
- i added a github/workflows folder with linters.yml file
- i added a index.html file with hello micorverse message
- i added a style.css file
- i added a header class and designed it further in the style.css file
- i added a .gitignore file
 microverse project from my feature branch "githubworkflows" into my master branch "main/Main-2022-09-22"

187jjay187 committed:
added index.html file with h1 tag saying Hello Microverse

187jjay187 committed:
added style.css file with my favourite color

187jjay187 committed:
added a .gitignore file to ignore the test.md hint-report and node_modules files

187jjay187 committed:
added github/workflows folder with linters.yml file as well as a README.md , MIT.md , app_screenshot.png , .hintrc , stylelintrc.json

187jjay187 commented:
I would like to merge this code for my hello microverse project from my feature branch "githubworkflows" into my master branch "main/Main-2022-09-22"

frank1738 commented on README.md:
```diff
@@ -1,2 +1,49 @@
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
- [ ] Your link to the MIT license is broken. `Hint`, replace  `LICENCE` with `MIT.md`. Make sure you customize the `MIT.md` file with your personal  info `(where required)`. This would make your readme file more professional

187jjay187 replied:
@frank1738 thank you for the review, I've added a description to my pull request and I've Fixed the broken link in my README.md file, please let me know if this is correct

frank1738 reviewed (changes requested):
Hi @187jjay187 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![YesYouAreGIF (2)](https://user-images.githubusercontent.com/50941074/172881298-64e98170-87c5-4075-8a77-71d65d991f55.gif)


## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay


## Required Changes ♻️
- [ ] Kindly add a description to your PR describing what you did in this project. This is important for documentation purposes. Your PR has no description

![image](https://user-images.githubusercontent.com/98453979/191852669-690add1c-ef3f-4f43-a598-92b8751b7bfb.png)

### Here is an example of a descriptive PR

![image](https://user-images.githubusercontent.com/98453979/191852781-20dc2d9f-a2e0-4a09-9c30-88aa81ff0228.png)




_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

187jjay187 committed:
added modified README.md with fixes to my MIT.md link

187jjay187 committed:
customised my MIT.md File

187jjay187 commented:
@frank1738 thank you for the review, I've added a description to my pull request and I've Fixed the broken link in my README.md file, please let me know if this is correct

jerryowusu commented on README.md:
```diff
@@ -1,2 +1,49 @@
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
[OPTIONAL] This should contain the link to the issues page of your repository. Kindly update it to make your project look more proffessional

187jjay187 replied:
Thank you for the insight into these issues, i really do appreciate it, i have resolved it

jerryowusu commented on README.md:
```diff
@@ -1,2 +1,49 @@
+
+
+
+## Getting Started
+
+**This Project was to make sure i understand the basic tools and methods i will be using on a daily basis during the course of my microverse programm as well as future work life.**
+
+
```
[OPTIONAL] Here, you are suppose to give information on how to clone the project, how to start or open the app, and what technologies or devices are needed. It should be something similar to the image below.

![Screenshot from 2022-09-23 08-52-35](https://user-images.githubusercontent.com/65233860/191925238-3c6e01a5-14e6-4298-937f-9074797e93c1.png)

187jjay187 replied:
Thank you for the further insight into these issues, I really do appreciate it, I have resolved them

jerryowusu reviewed (approved):
Hi @187jjay187 :wave:, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![clapping-leonardo-dicaprio](https://user-images.githubusercontent.com/65233860/191925741-47d8f21b-ce61-44ad-b560-77600adcd65a.gif)

## Highlights
:heavy_check_mark: All linters are passing
:heavy_check_mark: Good commit messages
:heavy_check_mark: Neat and well organized code

## Optional suggestions
[NOTE :warning: ] I was about to dissaprove your project and require changes when I saw you pushing the newly updated changes. Please do not submit your work for review untill you have finish making all the needed changes and adjustments to avoid any inconveniences in subsequent projects.  

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
